extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -976,12 +985,27 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
       return { error: "Incorrect Password" };
     }
 
+    let mnemonicPhrase = await getEncryptedTemporaryData({
```

Comment:
same change as the last bugfix: if we don't have the mnemonic, let's log in and get it